### ISSUE TITLE
Issues with Multi-geo tenant permissions

### ISSUE THREAD

**Issue by NadezhdaBzhilyanskaya** [SharePoint/sp-dev-docs#9733]
### What type of issue is this?

Question

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Target SharePoint environment

SharePoint Online

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- SPFx version 1.18.2
- Node.js version 18
- pnp/sp 4.0.1

### Issue description

This Issue is similar to [https://github.com/pnp/pnpjs/issues/1382](url) but unlike that issue I am trying to use the most recent version pnp/sp (4) so web.configure and  sp.setup are not longer functions that exists. 

TLDR: I am trying to update a site page's metadata where the files (specifically site pages) I am updating is on a different geo i.e. I am on https://tenant.sharepoint.com/ but trying to update https://tenantdeu.sharepoint.com/. They have the same tenant id but I get permission denied when attempting. I am able to update things on the geo I am currently on and I have tested this on both geos but just doesn't update across geos. 
So,
- [X]   https://tenant.sharepoint.com/  -> https://tenant.sharepoint.com/ 
- [X]   https://tenantdeu.sharepoint.com/  -> https://tenantdeu.sharepoint.com/ 
- [ ]   https://tenant.sharepoint.com/  -> https://tenant.sharepointdeu.com/ 
- [ ]   https://tenantdeu.sharepoint.com/  -> https://tenant.sharepoint.com/ 

I have tried this through both pnp/sp and through the graph client.  pnp/sp will get and update the current geo but won't get or update the other geo, while the graph client is able to grab both but will update neither.

I have tried adding permission to my package_soution file and still no luck.

Most solutions that I am seeing are requiring a app client and secret key which I do not want to resort to. 

Does anyone ave any experience with this and can suggest something I have not yet tried?

**Comment by Amey-MSFT** [comment]
Hello @NadezhdaBzhilyanskaya,
Thank you for bringing this issue to our attention.
Could you please confirm whether the issue is still occurring on your end?

**Comment by Amey-MSFT** [comment]
We are closing this issue for now. If the problem persists, feel free to reopen it or open a new one.
